oozie/src/main/java/org/apache/falcon/oozie/process/ProcessExecutionWorkflowBuilder.java

```diff
@@ -65,6 +65,8 @@
     private static final Set<String> FALCON_PROCESS_HIVE_ACTIONS = new HashSet<String>(
         Arrays.asList(new String[]{PREPROCESS_ACTION_NAME, USER_ACTION_NAME, }));
 
+    private static final String WF_LIB_SEPARATOR = ",";
```

Comment:
"WF_LIB_SEPARATOR" has been defined multiple times in the patch, can we define once it in some common utility class and use it across.